fybrik/README.md

```diff
@@ -0,0 +1,47 @@
+# Accessing a Dataset by a Fybrik Application
+
+We explain how to access a dataset through the Airbyte Module, by a Fybrik application. In this example, we use the `letter-frequency` dataset, a CSV file found in https://people.sc.fsu.edu/~jburkardt/data/csv/letter_frequency.csv.
```

Comment:
We explain how, using an Airbyte FybrikModule, a workload can access data stored in google-sheets, postgres, and other data stores supported by Airbyte connectors.  To do so a FybrikApplication (i.e. the request) must be submitted indicating the desired data set(s).  In this example ...